tests/integration/activities.test.ts

```diff
@@ -0,0 +1,95 @@
+import supertest from "supertest";
+import httpStatus from "http-status";
+
+import app, { init } from "../../src/app";
+import { clearDatabase, endConnection } from "../utils/database";
+import { createBasicSettings } from "../utils/app";
+import { createUser } from "../factories/userFactory";
+import {createLocation} from "../factories/locationFactory";
+import {createActivity} from "../factories/activityFactory";
+import { array } from "joi";
+
+const agent = supertest(app);
+
+beforeAll(async () => {
+  await init();
+});
+
+beforeEach(async () => {
+  await clearDatabase();
+  await createBasicSettings();
+});
+
+afterAll(async () => {
+  await clearDatabase();
+  await endConnection();
+});
+async function signIn(){
+    const user = await createUser();
+    const {email} = user
+    const session = await agent.post("/auth/sign-in").send({email,password:"123456"});
```

Comment:
Insira essa função numa factory e substitua o trecho `agent.post("/auth/sign-in").send({email,password:"123456"});` por um método nativo(metodos que o typeorm oferece como: save, find e etc) da sua entidade. Observação: utilize as rotas diretamente apenas se for testa-la.